src/components/Avatar.js

```diff
@@ -42,8 +43,12 @@ class Avatar extends PureComponent {
 
         const imageStyle = [
             this.props.size === 'small' ? styles.avatarSmall : styles.avatarNormal,
+
+            // Adding background makes RoomAvatar rectangular so adding background only if not chatRoom
```

Comment:
What do you think of this instead?
```suggestion
            // Background color isn't added for room avatar because it changes it's shape to a square
```